js/src/components/vertical-gap-layout/index.js

```diff
@@ -13,6 +13,15 @@ const sizeClassName = {
 	overlap: 'gla-vertical-gap-layout__overlap',
 };
 
+/**
+ * Renders a wrapper layout to make its children to be aligned vertically with a specified gap.
+ *
+ * @param {Object} props React props.
+ * @param {string} [props.className] Additional CSS class name to be appended.
+ * @param {''|'large'|'overlap'} [props.size=''] Indicate the gap between children. The empty string as default value is a normal gap.
+ *     'large': A larger gap.
```

Comment:
💅 🤔  Think it might be good to specify the value for normal gap and large gap.

Reply:
Adjusted by https://github.com/woocommerce/google-listings-and-ads/pull/968/commits/9baa3ebae5eda8aebcb0e528aa4d408472dac26e.